src/mainmenu.asm

```diff
@@ -746,13 +746,27 @@ EventsMenu:
     dw #events_goto_bosses
     dw #events_zebesawake
     dw #events_maridiatubebroken
+    dw #events_chozoacid
```

Comment:
I like this event, but not sure what the metroid room events do, those might just be wasting space

Reply:
Once you've killed all of the metroids in a room, they will no longer spawn. Only way to get them back is to load a preset (which would change your ammo and other events) or reset all events.

I have the Zebetite flags in my custom builds, but I left those out of the pull because I haven't figured out how the 4th zeb works.